### DISCUSSION TITLE
Hooks repository and project.

### DISCUSSION THREAD

**Discussion by dgduncan** [mochi-mqtt/server#318] [Ideas]
I wanted to float the possibility of creating a new repository and project called `hooks` or `hooks-extended` or something similar that contains hooks that are more "experimental" or a bit more specific. 

I currently have a repository in my account [here](https://github.com/dgduncan/mochi-cloud-hooks) that contains hooks that I use that would act as an inspiration. Obviously it would look very different than what I just linked. The idea behind this is to separate out the hooks that are more around building the ecosystem around `mochi-server` and not relevant for function of the server itself. Ideally the core server functionality is kept as minimal as possible.

**Comment by IshanDaga** [comment]
Seconded! Nice way of showcasing extensibility

**Comment by mochi-co** [comment]
Great idea - sounds good to me! 

I've created https://github.com/mochi-mqtt/hooks - let me know if you need me to make any changes 👍🏻
